src/app/blog/blog.controller.ts

```
@@ -0,0 +1,55 @@
+        }
+    }
+
+    static async deleteBlog(req: Request, res: Response): Promise<void> {
+        try {
+            await BlogService.deleteBlog(req.params.id);
+            res.status(200).json({ message: 'Blog deleted successfully' });
+        } catch (error) {
+            res.status(404).json({ error: (error as Error).message });
+        }
+    }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)